R/backend-mssql.R

```diff
@@ -144,24 +144,42 @@ mssql_temp_name <- function(name, temporary){
                             analyze = TRUE, ...) {
   NextMethod(
     table = mssql_temp_name(table, temporary),
-    temporary = FALSE
+    values = values,
+    overwrite = overwrite,
+    types = types,
+    temporary = FALSE,
+    unique_indexes = unique_indexes,
+    indexes = indexes,
+    analyze = analyze,
+    ...
   )
 }
 
 #' @export
 `db_save_query.Microsoft SQL Server` <- function(con, sql, name,
                                                  temporary = TRUE, ...){
-  NextMethod(
-    name = mssql_temp_name(name, temporary),
-    temporary = FALSE
+  name <- mssql_temp_name(name, temporary)
```

Comment:
Is there some reason you can no longer use `NextMethod()` here?

Reply:
The syntax is different in MSSQL. Originally implemented in #50, f283e03 reverted to using the parent method.

https://stackoverflow.com/q/16683758/946850

Adding a comment to the code.